app/services/notify/helpers/group-expenses-by-type.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (claimExpenses) {
```

Comment:
Used in 'cost-or-variance-equal-or-less-than-first-time-claim' and 'do-expenses-match-first-time-claim'